spec/unit/hanami/cli/commands/app/db/structure/load_spec.rb

```diff
@@ -88,23 +88,97 @@ def db_structure_dump
       db_structure_dump
     end
 
-    def table_exists?(slice, table_name)
-      slice["db.gateway"].connection
-        .fetch("PRAGMA table_info(#{table_name})")
-        .to_a.any?
-    end
```

Comment:
Being able to replace these with database-specific methods with a single `Hanami.app["db.gateway"].connection.tables.include?(:posts)` invocation is so much nicer. I wish I had explored Sequel's API a little more before I wrote these methods in the first place!